### DISCUSSION TITLE
Safari-like transparency

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [zen-browser/desktop#1330] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

It would be lovely if some elements of the UI had the frosted transparency effect like Safari, userChrome/userContent allows for this iirc, and it even supports backdrop blur on these elements. I only tested this in Firefox with my old rice on Linux so idk about other operating systems.

In Firefox on Linux you could just add rgba as background to navbar and the blur is provided by the compositor like picom.
If MacOS has a compositor with provided blur, then it might work too. Problem is that if you have blur disabled in picom, it will just make the elements transparent.

### Version

1.0.0-a.37

### What platform are you seeing the problem on?

macOS - Intel

**Comment by mr-cheffy** [comment]
Glad you liked it!

but...
>  I am not filing an enhancement request.

Ill transfer it into a discussion, but just so you know
